app/models/board.rb

```diff
@@ -13,4 +13,18 @@ def destroy_board
     self.destroy
   end
 
+  def self.create_new_board name
+    board = Board.new name: name
+
+    template = Board.first
+
+    template.cards.each do |card|
+      board.cards << (Card.create text: card.text, emoji: card.emoji)
```

Comment:
This shouldn't end up being a problem, but in theory we would want to do this as part of a transaction, to prevent creating a bunch of cards when the board itself doesn't get saved due to validation issues (and then you have orphaned cards).